packages/plugin-react/src/index.ts

```diff
@@ -366,5 +366,8 @@ function loadPlugin(path: string): Promise<any> {
 }
 
 // overwrite for cjs require('...')() usage
-module.exports = viteReact
-viteReact['default'] = viteReact
+// The following lines are inserted by scripts/patchEsbuildDist.ts,
+// this doesn't bundle correctly after esbuild 0.14.4
+//
+// module.exports = viteReact
+// viteReact['default'] = viteReact
```

Comment:
these 2 lines should not be marked as comments, right?

Reply:
No, as the comment above says, these are the two lines that are inserted after esbuild bundles the code. We can no longer add them before.